File: apps/web/src/lib/legal-links.ts
// Public legal documents are hosted on the publisher's site, not in this
// app -- keep these URLs in sync with the mobile app's AppConfig.
const LEGAL_BASE = "https://mustafaoner.net/kvkk-veri-isleme-gizlilik-politikalari/mk-adisyon";

export const LEGAL_LINKS = {
  terms: `${LEGAL_BASE}/kullanim-kosullari`,
  privacy: `${LEGAL_BASE}/gizlilik`,
  kvkk: `${LEGAL_BASE}/kvkk`,
} as const;

File: apps/mobile/lib/core/config/legal_links.dart
/// Public legal documents hosted on the publisher's site. Keep in sync
/// with apps/web/src/lib/legal-links.ts.
class LegalLinks {
  const LegalLinks._();

  static const String _base =
      'https://mustafaoner.net/kvkk-veri-isleme-gizlilik-politikalari/mk-adisyon';

  static final Uri privacy = Uri.parse('$_base/gizlilik');
  static final Uri terms = Uri.parse('$_base/kullanim-kosullari');
  static final Uri kvkk = Uri.parse('$_base/kvkk');
}
